packages/frontend/src/components/SiteWideBanner/index.tsx

```diff
@@ -1,28 +1,51 @@
-import { useCallback, useEffect, useState } from 'react'
+import { useCallback, useContext, useEffect, useState } from 'react'
 import CloseIcon from '@mui/icons-material/Close'
 import { Box, IconButton } from '@mui/material'
+import { LaunchDarklyContext } from 'contexts/LaunchDarkly'
 import { getItemForSession, setItemForSession } from 'helpers/storage'
-import useFormatMessage from 'hooks/useFormatMessage'
 
-const BANNER_TEXT_ID = 'bannerText'
+const LAUNCH_DARKLY_BANNER_KEY = 'app_banner_display'
```

Comment:
I think we can drop the `app_` prefix, I was intending for that to demarcate app actions & triggers. Just `banner_display` is enough. Or `banner_message` also can

Reply:
agreed